src/atomic_ops/sysdeps/gcc/mips.h

```diff
@@ -41,11 +41,19 @@
 
 # if !defined(_ABI64) || _MIPS_SIM != _ABI64
 #   define AO_T_IS_INT
+# if defined(__mips_isa_rev) && (__mips_isa_rev >= 6)
```

Comment:
\#if defined(x) && (x >= c)
=> 
\#if x >=c

See https://stackoverflow.com/questions/5085392/what-is-the-value-of-an-undefined-constant-used-in-if
Same for the below.